dataSources/Trade.csv

```diff
@@ -1,9 +1,65 @@
-TECHNOLOGY,INFUEL,IAR,OUTFUEL,OAR,MODE,CAPACITY (GW),DECOMISSION YEAR,CAPEX,Fixed O&M,Variable O&M,Fuel
-TRNCANMWCANWS,ELCCANMW02,1.05,ELCCANWS01,1,1,5.07,2060,3533,0,0,0
-TRNCANMWCANWS,ELCCANWS02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x
-TRNCANMWCANOT,ELCCANMW02,1.05,ELCCANOT01,1,1,9.46,2060,5037,0,0,0
-TRNCANMWCANOT,ELCCANOT02,1.05,ELCCANMW01,1,2,x,x,x,x,x,x
-TRNCANOTCANQC,ELCCANOT02,1.05,ELCCANQC01,1,1,17.9,2060,6064,0,0,0
-TRNCANOTCANQC,ELCCANQC02,1.05,ELCCANOT01,1,2,x,x,x,x,x,x
-TRNCANATCANQC,ELCCANAT02,1.05,ELCCANQC01,1,1,13.35,2060,4788,0,0,0
-TRNCANATCANQC,ELCCANQC02,1.05,ELCCANAT01,1,2,x,x,x,x,x,x
+TECHNOLOGY,INFUEL,IAR,OUTFUEL,OAR,MODE,CAPACITY (GW),DECOMISSION YEAR,CAPEX,Fixed O&M,Variable O&M,Fuel,Distance,StClairFunc
+TRNCANMWCANWS,ELCCANMW02,1.05,ELCCANWS01,1,1,5.07,2060,3533,0,0,0,x,x

```

Comment:
As mentioned, the lack of Distance and Saint Clair Functions might be a mistake here

Reply:
created issue